source/includes/openstack/_instances.md

```diff
@@ -135,6 +135,10 @@ Required | &nbsp;
 `networkId`<br/>*UUID* | The network in which the instance will be created. If you don't have a network, it can be created through the create network API.
 `securityGroupNames`<br/>*Array[string]* | The list of [security groups](#security-groups) to be associated to the instance
 
+Optional | &nbsp;
+------ | -----------
+`userData`<br/>*string* | Data that the user can specify when they launch an instance. User data holds a file that cloud-aware applications in the guest instance can access. One application that uses user data is the cloud-init system, which is available on various Linux systems.
```

Comment:
How is: "Data that the user can specify when they launch an instance. User data can be used in the cloud-init system as a script that is run during the initial launch. The user data field can be used to configure SSH keys, user settings, or run a script."